SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md

```diff
@@ -2,7 +2,7 @@ The CBL-Mariner SPEC files originated from a variety of sources with varying lic
 
 | Origin    | License                           |   SPECS    |
 |-----------|-----------------------------------|------------|
-| Fedora    | [Fedora MIT License Declaration](https://fedoraproject.org/wiki/Licensing:Main?rd=Licensing#License_of_Fedora_SPEC_Files)  | CUnit <br> Cython <br> ant-contrib <br> atf <br> attr <br> autoconf213 <br> babeltrace2 <br> brotli <br> byacc <br> calamares <br> cairo <br> catch <br> cpprest <br> cryptsetup <br> dnf <br> dpdk <br> docbook5-schemas <br> dos2unix <br> dwz <br> ebtables <br> extra-cmake-modules <br> gnu-efi <br> hardening-check <br> ipmitool <br> javapackages-tools <br> kde-settings <br> kexec-tools <br> kf5 <br> kf5-kconfig <br> kf5-kcoreaddons <br> kf5-ki18n <br> kf5-kwidgetsaddons <br> kpmcore-3.3.0 <br> kyua <br> leveldb <br> libatasmart <br> libburn <br> libcgroup <br> libcomps <br> libdnf <br> libisoburn <br> libisofs <br> libpwquality <br> libstoragemgmt <br> lua <br> lutok <br> mailcap <br> mokutil <br> mozjs60 <br> oath-toolkit <br> open-vm-tools <br> openssl <br> p11-kit <br> perl <br> perl-Archive-Zip <br> perl-Compress-Bzip2 <br> pkgconf <br> pycairo <br> pygobject3 <br> python-google-auth <br> python3 <br> python-jwt <br> python-kubernetes <br> python-pywbem <br> qt5-qtbase <br> qt5-qtdeclarative <br> qt5-qtsvg <br> qt5-qttools <br> qt5-rpm-macros <br> rdma-core <br> sgml-common <br> squashfs-tools <br> websocketpp <br> words <br> xmlstarlet <br> yaml-cpp <br> yasm <br> zipper |
+| Fedora    | [Fedora MIT License Declaration](https://fedoraproject.org/wiki/Licensing:Main?rd=Licensing#License_of_Fedora_SPEC_Files)  | CUnit <br> Cython <br> ant-contrib <br> atf <br> attr <br> autoconf213 <br> babeltrace2 <br> brotli <br> byacc <br> calamares <br> cairo <br> catch <br> cpprest <br> cryptsetup <br> dnf <br> dpdk <br> docbook5-schemas <br> dos2unix <br> dwz <br> ebtables <br> extra-cmake-modules <br> fribidi <br> gnu-efi <br> hardening-check <br> ipmitool <br> javapackages-tools <br> kde-settings <br> kexec-tools <br> kf5 <br> kf5-kconfig <br> kf5-kcoreaddons <br> kf5-ki18n <br> kf5-kwidgetsaddons <br> kpmcore-3.3.0 <br> kyua <br> leveldb <br> libatasmart <br> libburn <br> libcgroup <br> libcomps <br> libdnf <br> libisoburn <br> libisofs <br> libpwquality <br> libstoragemgmt <br> lua <br> lutok <br> mailcap <br> mokutil <br> mozjs60 <br> oath-toolkit <br> open-vm-tools <br> openssl <br> p11-kit <br> perl <br> perl-Archive-Zip <br> perl-Compress-Bzip2 <br> pkgconf <br> pycairo <br> pygobject3 <br> python-google-auth <br> python3 <br> python-jwt <br> python-kubernetes <br> python-pywbem <br> qt5-qtbase <br> qt5-qtdeclarative <br> qt5-qtsvg <br> qt5-qttools <br> qt5-rpm-macros <br> rdma-core <br> sgml-common <br> squashfs-tools <br> websocketpp <br> words <br> xmlstarlet <br> yaml-cpp <br> yasm <br> zipper |
```

Comment:
Could you also mention in the PR's summary why is we're adding/upgrading these specs. I'm not saying we shouldn't, it is just shrouded in mystery at the moment.;)

Reply:
Have mentioned the reason